website/src/index.css

```diff
@@ -149,7 +156,33 @@ a[target="_blank"] {
   background-repeat: no-repeat;
 }
 
+#root {
+  min-height: 100vh;
+}
+
+section {
+  min-height: calc(100vh - 6em - 2rem - 3px);
+}
+
 @media only screen and (width >= 768px) {
+  footer {
```

Comment:
the footer is unstyled for mobile, please move some of the styling there.

Reply:
![grafik](https://github.com/Idrinth/api-bench/assets/7874631/857c6b5e-86a0-49e3-b258-414daffaba00)

Reply:
Not sure what you mean do I have to create a media query for mobile version.

Reply:
no, mobile is the media-query-less default